Makefile.am

```diff
@@ -18,3 +18,9 @@ bootstrap: maintainer-clean
 
 test: all
 	make -C tests check-TESTS
+
+test-libyaml-test: libyaml-test
```

Comment:
So in addition to a better name here, could we also have a target that runs `test` and this `target` for simplicity's sake?